tests/client/amo/containers/TestSearch.js

```diff
@@ -9,19 +9,20 @@ import * as api from 'core/api';
 
 describe('Search.mapStateToProps()', () => {
   const state = {
+    api: { lang: 'fr-CA' },
     addons: { ab: { slug: 'ab', name: 'ad-block' },
              cd: { slug: 'cd', name: 'cd-block' } },
     search: { query: 'ad-block', loading: false, results: [{ slug: 'ab', name: 'ad-block' }] },
   };
 
   it('passes the search state if the URL and state query matches', () => {
     const props = mapStateToProps(state, { location: { query: { q: 'ad-block' } } });
-    assert.strictEqual(props, state.search);
+    assert.deepEqual(props, { lang: 'fr-CA', ...state.search });
```

Comment:
I changed this as it was erroring after I added the lang in here... even though the output looked the same. I hope this is alright, but if not let me know.